lib/fizzy/value.hpp

```diff
@@ -0,0 +1,20 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2019-2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <cstdint>
+
+namespace fizzy
+{
+union Value
```

Comment:
Can you add some basic unit tests in this commit? It is not very useful now but will be when adding floating point.

Reply:
Added.